Library/Homebrew/os/mac.rb

```diff
@@ -36,6 +36,26 @@ def locate tool
       end
     end
 
+    def install_name_tool
+      if File.executable?(path = "#{HOMEBREW_PREFIX}/opt/cctools/bin/install_name_tool")
+        Pathname.new(path)
+      else
+        locate("install_name_tool")
+      end
+    end
+
+    def otool
+      if File.executable?(path = "#{HOMEBREW_PREFIX}/opt/cctools/bin/otool")
+        Pathname.new(path)
+      else
+        locate("otool")
+      end
+    end
+
+    def can_build?
+      Xcode.installed? || CLT.installed?
+    end
```

Comment:
Sounds good to me! My only concern is that it might make these suffix conditionals very long - if that isn't a problem, I'll go ahead :smile: